src/Components/Articles.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+export default function Articles() {
+  const [articleList, setArticleLsit] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://nc-news-6m81.onrender.com/api/articles")
+      .then((response) => {
+        setArticleLsit(response.data);
+      });
+  }, []);
+  return (
+    <div>
```

Comment:
I'd consider adding in some visual feedback for the user if data is still loading 👀